public/src/components/switchboard.tsx

```diff
@@ -82,6 +99,28 @@ const useStyles = makeStyles(({ palette, spacing }: Theme) => ({
   textParagraph: {
     marginBottom: spacing(2),
   },
+  input: {
+    display: 'inline-block',
```

Comment:
I don’t really know what the `inline-block` display mode does: what’s the reason for using it here?